module/Crud/src/Controller/CrudController.php

```diff
@@ -0,0 +1,136 @@
+<?php
+declare(strict_types=1);
```

Comment:
Please add blank line before declare statement
We use PSR-12
Refer https://www.php-fig.org/psr/psr-12/